package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "benefiaciary-registration-portal",
```

Comment:
Let's rename this package to `benefiaciary-registration-portal-frontend` or `benefiaciary-registration-portal-ui`

Reply:
Sure